test/unit/helpers/application_helper_test.rb

```diff
@@ -43,4 +43,18 @@ class ApplicationHelperTest < ActionView::TestCase
       assert_in_delta(stats_graph_meter(@rubygem, most_downloaded_count), 12.5)
     end
   end
+
+  context "flash_message with html" do
+    setup do
+      @message = "This is a <strong>test</strong>"
+    end
+    
+    should "sanitize html" do
+      assert flash_message(:notice_html, @message).is_a?(ActiveSupport::SafeBuffer)
+    end
+
+    should "not sanitize non html" do
```

Comment:
Nit: this reads a little awkwardly in concert with the context - "flash_message with html > not sanitize non html". Perhaps the non-HTML case could be promoted to its own context?